### ISSUE TITLE
Metrics in verbose and metrics recorded from callbacks are different

### ISSUE THREAD

**Issue by swamika001** [keras-team/keras#10610]
Just run the "recording loss history" callback example from https://keras.io/callbacks/
Try it with any data you want, but but change verbose=0 to verbose=1
Except the first value, the loss values displayed by the verbose mode differ significantly from the loss values logged by the callback.
The same difference is also here when logging the accuracy
